x-pack/plugins/apm/server/lib/fleet/create_cloud_apm_package_policy.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  ElasticsearchClient,
+  SavedObjectsClientContract,
+  Logger,
+} from 'kibana/server';
+import {
+  APM_SERVER_SCHEMA_SAVED_OBJECT_TYPE,
+  APM_SERVER_SCHEMA_SAVED_OBJECT_ID,
+} from '../../../common/apm_saved_object_constants';
+import { APMPluginStartDependencies } from '../../types';
+import { getApmPackagePolicyDefinition } from './get_apm_package_policy_definition';
+
+export async function createCloudApmPackgePolicy({
+  fleetPluginStart,
+  savedObjectsClient,
+  esClient,
+  logger,
+}: {
+  fleetPluginStart: NonNullable<APMPluginStartDependencies['fleet']>;
+  savedObjectsClient: SavedObjectsClientContract;
+  esClient: ElasticsearchClient;
+  logger: Logger;
+}) {
+  const { attributes } = await savedObjectsClient.get(
+    APM_SERVER_SCHEMA_SAVED_OBJECT_TYPE,
+    APM_SERVER_SCHEMA_SAVED_OBJECT_ID
+  );
+  const apmServerSchema: Record<string, any> = JSON.parse(
+    (attributes as { schemaJson: string }).schemaJson
```

Comment:
type assertion for the saved object type `apm-server-schema`. Not sure if there's a better way to do this since data loses type information when being persisted via saved objects.